docs/folder-structure.md

```diff
@@ -0,0 +1,136 @@
+## Folder structure
+
+- `app/`
+
+  [Express](https://github.com/expressjs/express) application to preview components; also referred to as _Book a Secure Move Frontend_.
+
+  Each folder inside this directory refers to a `module`, which can contain
+  
+    - `module/index.js`
+       
+       Route configuration for the module
+       
+    - `module/controllers`
+    
+       A file or directory of files containing MVC controllers for the module
+       
+    - `module/fields`
+
+      Form wizard fields
+      
+    - `module/steps`
+    
+      Form wizard steps
+ 
+    - `module/views`
+    
+      Nunjucks page templates
+    - `module/validators`
+    
+       Form validators
+       
+    - `module/middleware`
+    
+       Middleware helpers to decorate the request with specific module data
+
+- `artifacts/` **contains auto-generated files**
```

Comment:
Do you think we should document other directories we gitignore, like `.build/`?

Reply:
Added in the `.build` and .circleci` directories.